### ISSUE TITLE
[BUG]: Innovation Backlog App - Initial Publish Error

### ISSUE THREAD

**Issue by anton3sco** [microsoft/powerapps-tools#1027]
**Describe the bug**
Imported the Innovation Backlog solution into Microsoft Dataverse for Teams. Everything went well except for when publishing the app (not the admin app) it came back with the error: The dependent component Entity (Id=duplicaterecord) does not exist

**Component (please tell us which flow or app you are experiencing issues with):**
 - Solution [e.g. CenterofExcellenceInnovationBacklog_1_30_managed]
 - Version [e.g. 1.3]
 - App or Flow [Innovation Backlog App]

**To Reproduce**
1. Import the solution
2. Run flows and configure as expected
3. Upload pictures to admin app
4. Open Innovation Backlog App and received the error

**Expected behavior**
I expected that the app would publish and post to the MS Teams channel selected. Instead, it gave an error screen shown in the screenshot. 

**Screenshots**
![image](https://user-images.githubusercontent.com/26906102/118058629-39022a00-b354-11eb-80d5-a5588cb391e2.png)

**Additional context**

**Comment by anton3sco** [comment]
Completed the install for the Innovation Backlog for Dataverse (Not for Teams) and that went as expected. I believe this gets me closer to thinking there may be a bug. Let me know if there is anything I can do to help! Thanks again!

**Comment by JeneferM-MSFT** [comment]
This sounds like the same type of error we see around environment variables. Can you tell me what else you have installed in that teams environment?

**Comment by anton3sco** [comment]
Hi Jenefer, This is the only thing I have installed on this Dataverse for Teams environment. It's a new environment. I could run through the installation on a new Teams if that would help troubleshoot things?

**Comment by JeneferM-MSFT** [comment]
No need. I repro. It looks like there may have been a product change around env vars here. I will investigate.

**Comment by JeneferM-MSFT** [comment]
Actually can you look to see if it did publish even though it errored? I see the same error as you but then it succeeds.

**Comment by anton3sco** [comment]
Very true! It did publish! Looks like it works as it should while quickly running through it. I ended up pursuing adding it to another Team while I waited and the same error there with the same result as it publishing behind the scenes 😂.

Question relevant to updating environment variables - I can do this in Dataverse no problem but in Dataverse for Teams it's not the same. Is there a magic trick to being able to modify the Environment Variables after import? I have read the directions here 100 times 😵🤪 https://docs.microsoft.com/en-us/power-platform/guidance/coe/setup-nurture-components#update-environment-variables 

I can open another ticket for the above if need be.

**Comment by manuelap-msft** [comment]
@anton3sco Regarding updating Environment Variables in Dataverse for Teams, we just noticed this as well - previously, there was a Default solution available in Teams environments and now that seems to be hidden. We are working with the Power Platform Engineering Team on an update. At the moment, it really does not seem possible to update Environment Variables from within Teams. Sorry about the inconvenience!

**Comment by manuelap-msft** [comment]
@anton3sco I have been given a workaround to update Environment Variables for Teams environments that you can use until a better option is available and hopefully that will unblock you in the meantime even though the steps are a bit complicated.

1. Go to flow.microsoft.com
2. Select your Teams environment from the Environment picker
3. Select Solutions
4. Select the Common Data Service Default Solution (should be listed at the very bottom)
![image](https://user-images.githubusercontent.com/55705669/118098286-9c7b6e80-b3cb-11eb-8fa8-6d918a4b415b.png)
5. Select + Add > Environment Variables
![image](https://user-images.githubusercontent.com/55705669/118098373-b917a680-b3cb-11eb-870b-ad21bd62d9a7.png)
6. Select the Environment Variable from the Center of Excellence - Core solution that you want to update and select Next and Add
![image](https://user-images.githubusercontent.com/55705669/118098413-c59bff00-b3cb-11eb-984d-abe1e475f316.png)
7. Once they have been added, selected Environment Variables in the new menu navigation 
8. Select the Environment Variable > Edit and update your default or current value
![image](https://user-images.githubusercontent.com/55705669/118098470-dba9bf80-b3cb-11eb-82cb-dbbd287c2c4d.png)

**Comment by manuelap-msft** [comment]
Hello,

we're in the process of moving the CoE Starter Kit from this repo (powerapps-tools) to it's own dedicated repo (coe-starter-kit). We are transferring issues to the new repo today, and you will receive responses from the new repo. The new issue URL is

Sorry for any inconvenience this may cause
Manuela
